test/cuda/test_tc_mapper.cc

```diff
@@ -297,6 +297,67 @@ def tensordot(float(N, C1, C2, H, W) I0, float(N, C2, C3, H, W) I1) -> (O) {
   ::benchmarkKernelOptions(TC, name, inputs, options);
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// TensorAdd2D -- Version 0
+//   O(n, m) +=! I0(n, m) * I1(n, m)
+///////////////////////////////////////////////////////////////////////////////
+TEST_F(TcCudaMapperTest, TensorAdd2Dv0) {
```

Comment:
this test does not really test anything interesting at this point, I would remove it